tests/integration-tests/tests/api/test_api_client/test_api_client/pcluster.config.yaml

```diff
@@ -0,0 +1,21 @@
+Image:
```

Comment:
as a reminder for a separate PR: it might be good to make this cluster config use as many features as possible so that we can better test the IAM policy attached to the API.